desktop/cmp/treemap/TreeMap.ts

```diff
@@ -111,9 +105,12 @@ class TreeMapLocalModel extends HoistModel {
 
     chart = null;
     clickCount = 0;
+    debouncedClickHandler;
+
+    _prevConfig;
```

Comment:
think we should go ahead and use private on any properties we can  -- but don't feel strongly about it.